code/modules/clothing/gloves/color.dm

```diff
@@ -40,6 +40,19 @@
 	if(user.mind?.assigned_role in GLOB.security_positions)
 		SEND_SIGNAL(user, COMSIG_CLEAR_MOOD_EVENT, "sec_insulated_gloves")
 
+/obj/item/clothing/gloves/color/yellow/attackby(obj/item/I, mob/user, params)
+	if(I.tool_behaviour == TOOL_KNIFE || I.is_sharp())
+		var/turf/T = get_turf(src)
+		var/obj/item/stack/sheet/insulated_fibers/F = new (T, 5)
+		if(QDELETED(F))
+			F = locate(/obj/item/stack/sheet/insulated_fibers/) in T
+		if(F)
+			transfer_fingerprints_to(F)
+			F.add_fingerprint(user)
+		qdel(src)
+		to_chat(user, "<span class='notice'>You tear [src] apart.</span>")
+	else
+		return ..()
```

Comment:
This and the code for ``/obj/item/clothing/gloves/color/fyellow`` is very similar. Do you think you could try minimizing the code duplication?

Reply:
How? Do I make it a subtype of the insulated ones? I'm still pretty new on coding so I don't know how to clean code to not be duplicated like this

Reply:
could have cutting gloves be a thing on all gloves, and give insulated and fake insulated gloves variables to give their respected drops